### DISCUSSION TITLE
首页Frontmatter 配置中features配置不生效问题

### DISCUSSION THREAD

**Discussion by gaozhao7** [vuepress-theme-hope/vuepress-theme-hope#3551] [Q&A] [answered]
这块配置为什么在首页没有任何展示呢？看文档中应该是支持，使用其他主题时有内容显示
<img width="834" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/36031132/99cc509e-7426-4dea-88c7-aa8382111807">

**Comment by Mister-Hope** [comment, marked as answer]
highlights has higher priority than features
